android/engine/src/main/java/org/smartregister/fhircore/engine/domain/model/TopScreenSectionConfig.kt

```diff
@@ -25,4 +25,5 @@ data class TopScreenSectionConfig(
   val searchBar: RegisterContentConfig?,
   val title: String? = null,
   val menuIcons: List<ImageProperties>? = null,
+  val shouldShowSearchButton: Boolean = false,
```

Comment:
```suggestion
  val showSearchButton: Boolean = false,
```

Better check if the `searchBar` configuration is `null`. If null do not display the search bar. The new property won't be necessary.

Reply:
This is for the extra button on searchBar like on Multi select widget which triggers the sync when clicked.